microsoft-edge/progressive-web-apps-edgehtml/get-started.md

```diff
@@ -314,124 +333,106 @@ The following is adapted from the *Push Rich Demo* in Mozilla's [Service Worker
-
-Check out our other PWA guides to learn how to increase customer engagement and provide a more seamless, OS-integrated app experience:  
+Of course, there is a lot more that goes into [making a great PWA][PwaEdgehtmlIndexRequirements] beyond what you read here, including responsive design, deep-linking, [cross-browser testing][BrowserStackTestEdgeBrowser] and other [best practices][Webhint] \(not to mention your app functionality!\), but hopefully this guide gave you a solid introduction of PWA basics and some ideas on getting started.  If you have further questions on PWA development with Windows or with Visual Studio, please leave a comment!  
 
-*   [**Windows tailoring.**][PwaEdgehtmlWindowsFeatures] Using simple feature detection, you can progressively enhance your PWA for Windows 10 customers through native Windows Runtime \(WinRT\) APIs, such as those for customizing Windows **Start** menu tile notifications and taskbar jumplists, and \(upon permission\) working with user resources, such as photos, music and calendar.  
+Check out the other PWA guides to learn how to increase customer engagement and provide a more seamless, OS-integrated app experience.  
 
-*   [**PWAs in the Microsoft Store.**][PwaEdgehtmlMicrosoftStore] Learn more about the benefits of app store distribution and how to submit your PWA.  
+*   [Windows tailoring][PwaEdgehtmlWindowsFeatures]. Using simple feature detection, you may progressively enhance your PWA for Windows 10 customers through native Windows Runtime \(WinRT\) APIs, such as those for customizing Windows **Start** menu tile notifications and taskbar jumplists, and \(upon permission\) working with user resources, such as photos, music, and calendar.  
+*   [PWAs in the Microsoft Store][PwaEdgehtmlMicrosoftStore].  Learn more about the benefits of app store distribution and how to submit your PWA.  
 
-Also be sure to explore the great resources from around the dev community!  [MDN web docs][MDNProgressiveWebApps] and [Google Developers][WebDevProgressiveWebApps] have excellent guides on Progressive Web Apps.  [https://pwa.rocks][ProgressiveWebApps] showcases real-world examples of PWAs, and [https://hnpwa.com][HackerNewsProgressiveWebApps] compares different frameworks and performance patterns for implementing a sample \(*Hacker News* reader\) PWA.  
+Also be sure to explore the great resources from around the dev community!  [MDN web docs][MDNProgressiveWebApps] and [Google Developers][WebDevProgressiveWebApps] have excellent guides on Progressive Web Apps.  [https://pwa.rocks][ProgressiveWebApps] showcases real-world examples of PWAs, and [https://hnpwa.com][HackerNewsProgressiveWebApps] compares different frameworks and performance patterns for implementing a sample \(Hacker News reader\) PWA.  
 
 <!-- image links -->  
 
```

Comment:
Can we include a See Also section with related links?

Reply:
It is possible. 
I will start a thread.

Reply:
See also implemented.